{"name": "test_on_asser_for_allure", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2", "trace": "def test_on_asser_for_allure():\n        \"\"\"Test on assert\"\"\"\n>       assert 1==2\nE       assert 1 == 2\n\nusers/test_user.py:51: AssertionError"}, "description": "Test on assert", "start": 1667511662115, "stop": 1667511662115, "uuid": "abd6ae98-2ff6-4e4a-a4ef-45eb91268efe", "historyId": "fb3388d55b03d45ead87749307362677", "testCaseId": "b374734fef6268171c6a2ba23731e0aa", "fullName": "tests.users.test_user#test_on_asser_for_allure", "labels": [{"name": "parentSuite", "value": "tests.users"}, {"name": "suite", "value": "test_user"}, {"name": "host", "value": "MacBook-Air-M1.Dlink"}, {"name": "thread", "value": "22453-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.users.test_user"}]}